src/Microsoft.Fast.Components.FluentUI/DataGridCellType.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Fast.Components.FluentUI
+{
+    public enum DataGridCellType
+    {
+        Default,
+        ColumnHeader
```

Comment:
fyi - there is a pr out to add a 'rowHeader' type that should be in shortly - https://github.com/microsoft/fast/pull/5206

Reply:
I'll add it on this side ahead of the fast side pr merge.